bindings/experimental/distrdf/python/DistRDF/Backends/Spark/Backend.py

```diff
@@ -161,3 +168,39 @@ def make_dataframe(self, *args, **kwargs):
         npartitions = kwargs.pop("npartitions", self.optimize_npartitions())
         headnode = HeadNode.get_headnode(npartitions, *args)
         return DataFrame.RDataFrame(headnode, self)
+
+    @staticmethod
+    def RunGraphs(proxies, numthreads=4):
+        """
+        Trigger multiple RDF graphs through multithreading, according to Spark
+        docs on job scheduling at https://spark.apache.org/docs/latest/job-scheduling.html#scheduling-within-an-application.
+
+        Args:
+            proxies(iterable): Action proxies that should be triggered. Only
+                actions belonging to different RDataFrame graphs will be
+                triggered to avoid useless calls.
+
+            numthreads(int, optional): Number of threads to spawn at the same
+                time. Each thread will submit a separate job to the Spark
+                cluster through the same SparkContext. Defaults to 4.
```

Comment:
Perhaps default 2? The lowest number that implies parallelism and is greater than 1.

Reply:
why don't we spawn as many threads as the number of event loops that need to be spawned? (i guess this is basically a discussion on what value `concurrent_runs` should have)

Reply:
Yes indeed. I imagined putting a default limit to `concurrentruns` could be a good idea in general, to avoid overloading the scheduler. If you don't agree I can just remove the queue logic and spawn as many jobs as action proxies

Reply:
What if I created 1000 histos, will I spawn 1000 threads? I think at least we should protect against high values.

Reply:
no, it's one thread per separate computation graph

Reply:
Ok then we need to group proxies with the same RDF head node, and one of them is given to one thread for the trigger.

Reply:
it's already done if I read the code correctly, grep for `uniqueproxies`

Reply:
Yes it's already done, (I used a similar logic to the c++ version). But what enric mentions is still true if for *any reason* somebody has 1000 separate histos coming from 1000 separate RDF

Reply:
i don't know if that's a realistic usecase, but anyway if they have the resources to back it up, why throttling them? does spark choke if a thousand jobs are submitted to the scheduler?

Reply:
In principle no, it just puts everything in a queue anyway (or whatever kind of scheduling the cluster has in place, so not anything we really control). This is just a way to tell Spark "I have lots of jobs I would like to process asap"